psi4/src/psi4/libmints/basisset.cc

```diff
@@ -192,24 +191,19 @@ void BasisSet::print(std::string out) const
 {
     std::shared_ptr <psi::PsiOutStream> printer = (out == "outfile" ? outfile :
                                                    std::shared_ptr<OutFile>(new OutFile(out)));
-    if(basistype_ == GaussianBasis) {
-        printer->Printf("  Basis Set: %s\n", name_.c_str());
-        printer->Printf("    Blend: %s\n", target_.c_str());
-        printer->Printf("    Number of shells: %d\n", nshell());
-        printer->Printf("    Number of basis function: %d\n", nbf());
-        printer->Printf("    Number of Cartesian functions: %d\n", nao());
-        printer->Printf("    Spherical Harmonics?: %s\n", has_puream() ? "true" : "false");
-        printer->Printf("    Max angular momentum: %d\n\n", max_am());
-    } else if(basistype_ == ECPBasis) {
-
+    printer->Printf("  Basis Set: %s\n", name_.c_str());
+    printer->Printf("    Blend: %s\n", target_.c_str());
+    printer->Printf("    Number of shells: %d\n", nshell());
+    printer->Printf("    Number of basis function: %d\n", nbf());
+    printer->Printf("    Number of Cartesian functions: %d\n", nao());
+    printer->Printf("    Spherical Harmonics?: %s\n", has_puream() ? "true" : "false");
+    printer->Printf("    Max angular momentum: %d\n\n", max_am());
+    if(has_ECP()) {
         printer->Printf("  Core potential: %s\n", name_.c_str());
-        printer->Printf("    Blend: %s\n", target_.c_str());
-        printer->Printf("    Number of shells: %d\n", nshell());
-        printer->Printf("    Number of terms: %d\n", nprimitive());
+        printer->Printf("    Number of shells: %d\n", n_ecp_shell());
+        printer->Printf("    Number of terms: %d\n", n_ecp_primitive());
```

Comment:
Yeah, not really sure what to call these.  We normally think of "primitives" as GTOs combined to approximate an STO.  That's not what's happening in ECPs, so I thought a different word would be better.  Happy to take suggestions on this